src/pages/trader-tools/common/_style.ts

```diff
@@ -536,6 +535,13 @@ export const FormulaHighlightMobile = styled(Flex)`
     font-size: 14px;
     font-weight: 600;
     flex-direction: ${(props) => (props.syn_mobile ? 'column' : '')};
+
+    @media ${device.tablet} {
+        width: 328px;
+    }
+    @media ${device.mobileM} {
+        width: 280px;
+    }
```

Comment:
Are we able to use `width: calc(100% - 32px)` or 100% instead? Depending if the parent wrapper has already accounted for the left right margins of 16px each.